### PR TITLE
added sample workload, logging and autoscaling sections

### PR DESCRIPTION
**Pull request by prabhatsharma**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by dalbhanj — APPROVED** [review]
LGTM

**Comment by dalbhanj** [comment]
https://github.com/aws-samples/eks-workshop/issues/1075

File: content/advanced/430_emr_on_eks/monitoring_and_logging_0.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: "Monitoring and logging  Part 1 - Setup"
+date: 2021-04-08T04:25:46-07:00
+weight: 30
+draft: true
```
Comment (dalbhanj) [inline content/advanced/430_emr_on_eks/monitoring_and_logging_0.md:5]: change draft to false to make page visible

File: content/advanced/430_emr_on_eks/monitoring_and_logging_1.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: "Monitoring and logging  Part 2 - Cloudwatch & S3"
+date: 2021-04-08T04:25:46-07:00
+weight: 30
+draft: true
```
Comment (dalbhanj) [inline content/advanced/430_emr_on_eks/monitoring_and_logging_1.md:5]: change draft to false to make page visible

File: content/advanced/430_emr_on_eks/monitoring_and_logging_2.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Monitoring and logging  Part 3 - Spark History server"
+date: 2021-04-08T04:25:46-07:00
+weight: 35
+draft: true
```
Comment (dalbhanj) [inline content/advanced/430_emr_on_eks/monitoring_and_logging_2.md:5]: change draft to false to make page visible

File: content/advanced/430_emr_on_eks/sample_workload.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
+title: "Run sample workload"
+date: 2021-04-08T03:21:44-07:00
+weight: 20
+draft: true
```
Comment (dalbhanj) [inline content/advanced/430_emr_on_eks/sample_workload.md:5]: change draft to false to make page visible

File: content/advanced/430_emr_on_eks/autoscaling.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+date: 2021-04-08T01:29:46-07:00
+weight: 40
+draft: false
+---
+
+Autoscaling in EKS is achieved using Cluster Autoscaler.
+
+When resources are not available EKS will add new nodes to the cluster to create new pods and execute the jobs.
+
+Refer to [Cluster autoscaler](beginner/080_scaling/deploy_ca/) section in order to configure the cluster autoscaler.
```
Comment (dalbhanj) [inline content/advanced/430_emr_on_eks/autoscaling.md:12]: Let's aim to give complete experience for workshop users. Be creative, think about simple and complex use case and think about what an analytics person should know about autoscaling. We need to add another chapter about DRA (dynamic resource allocation) as well

**Review by dalbhanj — COMMENTED** [review]
Hi @prabhatsharma , thanks for contribution. I left some feedback. Let me know if you have questions

File: content/advanced/430_emr_on_eks/autoscaling.md (hunk 5 above)
Comment (joebowbeer) [reply content/advanced/430_emr_on_eks/autoscaling.md:12]: Nit: Should clarify that this is vertical autoscaling?

**Pull request merged**